packages/design-system/src/components/Badge/Badge.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import styles from './style.scss';
+
+function Badge(props) {
+  const badgeType = `badge--${props.type || 'primary'}`;
+
+  return (
+    <span className={`${styles.badge} ${styles[badgeType]}`}>
+      {props.children}
+    </span>
+  );
+}
+
+Badge.propTypes = {
+  children: PropTypes.node.isRequired,
+  /**
+   * Type of badge
+   */
+  type: PropTypes.oneOf(['primary', 'light'])
```

Comment:
I would rather avoid using words like 'dark' or 'light'. We have started to develop themes in the design-system (and soon it will be used in AA) and this named can be problematic. I think the word `secondary` is much better here.

Reply:
Are you sure that we have only 3 types of badge? Please, ask @Jolama for more colors.
![image](https://user-images.githubusercontent.com/24878280/74143558-4f728080-4bfb-11ea-817b-54eaa74e4121.png)

Reply:
Did you ask the designer about used badges colors?

Reply:
@Neob91 ping

Reply:
Yes, this is something else than badge.